src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -141,7 +142,7 @@ If you need to change the default privacy settings, you can choose specific clas
   </Collapser>
   </CollapserGroup>
 
-* <DoNotTranslate>**Block**</DoNotTranslate> means content will be replaced with an empty placeholder. For example, if you have a video playing on your About Us webpage and don’t want it captured, you could block the class that contains that video.
+* <DoNotTranslate>**Block**</DoNotTranslate> means content will be replaced with an empty placeholder. For example, if you have an image on your About Us webpage and don’t need it captured, you could block the class containing the image.
```

Comment:
Is a screenshot the best example for the Block feature? Is there another one more relevant to customers?